app/views/pages/home.html.haml

```diff
@@ -4,7 +4,7 @@
   .govuk-width-container
     .govuk-grid-row
       .govuk-grid-column-two-thirds
-        .search_panel.location-search= render 'pages/home/search'
```

Comment:
There is a reason for the 2 different classes

Reply:
See the subscriptions new/edit pages and the home page location inputs for comparison. If everything was under `.location-search` then the everywhere that class is applied would look like the home page search panel